code/sql/algoliasearch_setup/mysql4-upgrade-1.5.0-1.6.0RC.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+/** @var Mage_Core_Model_Resource_Setup $installer */
+$installer = $this;
+
+$installer->startSetup();
+
+// Need to transform "removeProducts" to "rebuildProductIndex" as re-indexing was refactored and "removeProducts" do not exists anymore
+$installer->run("DELETE FROM `{$installer->getTable('algoliasearch/queue')}`");
```

Comment:
I changed the query? Do you have any idea what to do with the filename? ^ @maxiloc